.github/workflows/ci-ultra-tests.yml

```diff
@@ -44,6 +44,13 @@ jobs:
           --ignore=./tests/test_social_vehicles.py \
           --ignore=./tests/test_rllib_train.py \
           --ignore=./tests/test_tune.py \
+          --ignore=./tests/test_train.py \
+          --ignore=./tests/test_evaluate.py
+          pytest -v \
```

Comment:
Perhaps we can add a comment that explains why we have to divide up the heavy tests?

Reply:
Of course, that is indeed needed 👍